unverified/sexpr-bootstrap/x64/64/Holmakefile

```diff
@@ -18,7 +18,25 @@ cake-sexpr-$(ARCH)-$(WORD_SIZE): *$(ARCH)SexprScript.sml
 cake.S: cake-sexpr-$(ARCH)-$(WORD_SIZE)
 	$(CAKE_BIN) --sexp=true --exclude_prelude=true --skip_type_inference=true --emit_empty_ffi=true --heap_size=$(HEAP_MB) --stack_size=$(STACK_MB) --reg_alg=0 <$< >$@
 
-cake-unverified-$(ARCH)-$(WORD_SIZE).tar.gz: cake.S basis_ffi.c Makefile cake-sexpr-$(ARCH)-$(WORD_SIZE)
-	tar -chzf $@ --transform='s|^|cake-unverified-$(ARCH)-$(WORD_SIZE)/|' cake.S basis_ffi.c Makefile cake-sexpr-$(ARCH)-$(WORD_SIZE)
+cake-unverified-$(ARCH)-$(WORD_SIZE).tar.gz: cake.S basis_ffi.c Makefile cake-sexpr-$(ARCH)-$(WORD_SIZE) how-to.output
+	cp $(CAKEMLDIR)/how-to.md how-to.md
+	tar -chzf $@ --transform='s|^|cake-unverified-$(ARCH)-$(WORD_SIZE)/|' cake.S basis_ffi.c Makefile cake-sexpr-$(ARCH)-$(WORD_SIZE) how-to.md
+	rm -f how-to.md
 
-EXTRA_CLEANS = cake.S cake-sexpr-$(ARCH)-$(WORD_SIZE) cake-unverified-$(ARCH)-$(WORD_SIZE).tar.gz
+# the following extracts the code from how-to.md, compiles it and runs it
+
+how-to.output: how-to.cake
+	./how-to.cake > how-to.output
+
+how-to.cake: extract_code.cake $(CAKEMLDIR)/how-to.md cake
+	./extract_code.cake $(CAKEMLDIR)/how-to.md > how-to.cml
+	make how-to.cake
+	rm -f how-to.cml
+
+extract_code.cake: extract_code.cml cake
+	make extract_code.cake
+
+cake: cake.S basis_ffi.c
+	make cake
+
+EXTRA_CLEANS = cake.S cake-sexpr-$(ARCH)-$(WORD_SIZE) cake-unverified-$(ARCH)-$(WORD_SIZE).tar.gz cake extract_code.cake how-to.cml how-to.cake how-to.output
```

Comment:
these should be in an appropriate `.gitignore` too if they ever stick around. (maybe they are always immediately deleted, I am not sure.)

Reply:
I have added them to the .gitignore